<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFormClass</class>
 <widget class="QMainWindow" name="UserFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QListWidget" name="listWidgetIssues"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButtonAdd">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="pushButtonRemove">
      <property name="text">
       <string>Remove </string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="pushButtonResolve">
      <property name="text">
       <string>Resolve</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="UserForm.qrc"/>
 </resources>
 <connections/>
</ui>
